packages/components/src/components/hds/form/header/title.ts

```diff
@@ -13,7 +13,7 @@ import type { HdsTextDisplaySignature } from '../../text/display.ts';
 
 export const DEFAULT_SIZE = HdsTextSizeValues.FourHundred;
 export const DEFAULT_TAG = HdsFormHeaderTitleTagValues.Div;
-export const AVAILABLE_TAGS: string[] = Object.values(
+export const AVAILABLE_TAGS: HdsFormHeaderTitleTags[] = Object.values(
```

Comment:
```suggestion
export const TAGS: HdsFormHeaderTitleTags[] = Object.values(
```
[Note][non-blocker] For most of these exported values in components I don't think we use the "AVAILABLE_" prefix. I do see this prefix is used across a couple components such as the card container and form header. Maybe a future standardization effort could be done to update these to align to our pattern.